upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.23.yaml.template

```diff
@@ -4545,6 +4545,9 @@ spec:
               name: cni-net-dir
           securityContext:
             privileged: true
+        # temporary fix to avoid server side apply issues
+        - name: flexvol-driver
+          image: busybox
```

Comment:
```suggestion
        - name: flexvol-driver
          image: busybox
          command: ['sh', '-c', 'echo Temporary fix to avoid server side apply issues']
```